src/test_log_parser.py

```diff
@@ -0,0 +1,64 @@
+import unittest
```

Comment:
Our naming convention for unit test files is usually to append _test to the name, rather than prefixing test_. So it would be better to name this file log_parser_test.py.

Reply:
Good to know, I'll switch this for other tests as well.

Reply:
Ah, I didn't realize that these comments used formatting. My underscores turned into fun italics.